src/datastore/pg-store.ts

```diff
@@ -133,7 +136,7 @@ export class PgStore {
 
   async close(): Promise<void> {
     await this.notifier?.close();
-    await this.sql.end();
+    await this.sql.end({ timeout: this.closeTimeout });
```

Comment:
This line is the primary fix in this PR right?

Reply:
Yes, that's correct